modules/@apostrophecms/home-page/index.js

```diff
@@ -5,6 +5,29 @@ module.exports = {
   },
   fields: {
     add: {
+      cta: {
+        type: 'area',
+        contextual: true,
+        max: 1,
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {
```

Comment:
This seems like it should be a string field. It's not using RTE at all and RTE is requiring aggressive CSS.

Reply:
going to keep this for fun demo purposes